fdbserver/DataDistribution.actor.cpp

```diff
@@ -125,7 +125,7 @@ void DataMove::validateShard(const DDShardInfo& shard, KeyRangeRef range, int pr
 	ASSERT(!this->meta.ranges.empty() && this->meta.ranges.front().contains(range));
 
 	if (!shard.hasDest) {
-		TraceEvent(SevError, "DataMoveValidationError")
```

Comment:
Why do we change SevError to SevWarnAlways?

Reply:
We know this could happen in theory, and we can recover from it.